### ISSUE TITLE
yolov5x training problems

### ISSUE THREAD

**Issue by Caesar-xu7** [ultralytics/yolov5#11262]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

when I try to training my model, I account this warning. Meanwhile, mAP@0.5 , precition and recall is very very slow, nearly 0.
train.py:317: FutureWarning: Non-finite norm encountered in torch.nn.utils.clip_grad_norm_; continuing anyway. Note that the default behavior will change in a future release to error out if a non-finite total norm is encountered. At that point, setting error_if_nonfinite=false will be required to retain the old behavior.

### Environment

YOLOv5 🚀 v6.1-267-g50ff6eee Python-3.8.10 torch-1.9.0+cu111 CUDA:0 (NVIDIA RTX A5000, 24256MiB)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@Caesar-xu7 This warning is raised by PyTorch when it encounters non-finite gradient values during gradient clipping. The gradient clipping function (torch.nn.utils.clip_grad_norm_) is used in YOLOv5 to prevent gradients from becoming too large during training, which helps stabilize training and mitigate the exploding gradient problem.

The warning suggests that there are non-finite values in the gradients, such as NaN (Not a Number) or infinity, which might be caused by numerical instability during training. When non-finite gradient values are encountered, it can negatively impact the training process and lead to poor performance, such as low mAP, precision, and recall.
